cmd/vic-machine/configure/configure.go

```diff
@@ -325,6 +325,8 @@ func (c *Configure) Run(clic *cli.Context) (err error) {
 		log.Errorf("Configuring cannot continue - failed to create validator: %s", err)
 		return errors.New("configure failed")
 	}
+	defer validator.Session.Logout(ctx)
```

Comment:
Is this what you mean @emlin https://github.com/vmware/vic/pull/5857/commits/2440332e8421ce51726b54e6957db40444a26c55